autocapture.md

```diff
@@ -0,0 +1,36 @@
+This is a rough proposal that fits into the category of complex product decisions. I will polish this markdown as a PR.
+
+# Proposal: stop supporting autocapture
+
+We currently support autocapture of front end events.
+
+This helps us with:
+
+* Users can get events in more quickly
+* It captures lots of useful pageview data
+* Perhaps generates initial sign ups who we retain for _other_ reasons
+
+Hypothesis for why this is hurting us:
+
+Complexity.
+
+* It generates lots of auto labelled events, which will lead to confusing experiences for users or incorrect decision-making
+* It doesn't solve the whole problem - you still can't get email addresses for example with it.
+  * This makes the solution _more_ confusing: you now need to do manual events AND train users to use autocapture too
+* It makes mobile a 2nd class citizen, since it isn't supported with autocapture
```

Comment:
Support for React Native is possible, yet has been deprioritized. Heap does it though, and we could do the same. Context [here](https://github.com/PostHog/posthog/issues/1174) and [here](https://heap.io/blog/engineering/how-we-leveraged-asts-and-babel-to-capture-everything-on-react-native-apps).